src/print/mod.rs

```diff
@@ -340,3 +340,44 @@ pub fn json_item_to_string<I: FormatExt>(item: &I, config: &Config)
     prn.end().unwrap_exc()?;
     Ok(out)
 }
+
+pub fn prompt(line: &str) {
+    println!(
+        "{}",
+        line.bold().color(Color::Orange3),
+    );
+}
+
+pub fn error(line: &str) {
+    eprintln!(
+        "{}",
+        line.bold().light_red(),
+    );
+}
+
+pub fn error_msg(title: &str, msg: &str) {
+    eprintln!(
+        "{}: {}",
+        title.bold().light_red(),
+        msg.bold().white(),
+    );
+}
+
+pub fn error_obj(e: &dyn fmt::Display) {
+    error_msg("edgedb error", &format!("{:#}", e));
+}
```

Comment:
I think we wanted to prefix all error messages with `edgedb error` specifically, so that it's clear where error comes from in scripts. So I'm not sure what `error_msg` can be used for later. And why `error` doesn't have this prefix.

Reply:
Added `edgedb error` for all errors - and removed the `error_msg()` function.